data/protocols/250/0xDAO.json

```diff
@@ -17,7 +17,7 @@
     },
     "de": {
       "name": "0xDAO",
-      "description": "0xDAO is a protocol that is pooling money to distribute a Solidly NFT's token emissions via their governance token OXD."
+      "description": "0xDAO ist ein Protokoll, das Geld zusammenlegt, um die Token-Emissionen von Solidly NFT über ihren Governance-Token OXD zu verteilen."
```

Comment:
why is protocol geld token-emissions and governance token capitalized?

Reply:
They are captitalized because all nouns need to be capitalized in in German.

Reply:
ok thanks